test/e2e/settings/proxy_protocol.go

```diff
@@ -41,16 +39,19 @@ var _ = framework.IngressNginxDescribe("Proxy Protocol", func() {
 	BeforeEach(func() {
 		err := f.NewEchoDeployment()
 		Expect(err).NotTo(HaveOccurred())
+
+		err = f.UpdateNginxConfigMapData(setting, "false")
+		Expect(err).NotTo(HaveOccurred())
```

Comment:
Why is this necessary? I could not find the corresponding `updateConfigmap` for this

Reply:
It's happening in the AfterEach handler in the current implementation: https://github.com/kubernetes/ingress-nginx/pull/2346/files#diff-271db20be8cbad3b37eafe10f4eacbcfL47

It seems like the test writer just wanted to default the setting to `false` on all tests, and explicitly set `true` at the beginning of tests where needed.